cohorts/survival.py

```diff
@@ -72,59 +77,89 @@ def plot_kmf(df,
         if is_median:
             label_suffix += " (median)"
         default_label_with_condition = "%s > %s" % (condition_col, label_suffix)
-    else:
+        with_condition_label = with_condition_label or default_label_with_condition
+        no_condition_label = no_condition_label or default_label_no_condition
+        label_map = {False: no_condition_label,
+                     True: with_condition_label}
+        color_map = {False: no_condition_color,
+                     True: with_condition_color}
+    elif df[condition_col].dtype == 'O' or df[condition_col].dtype.name == "category":
```

Comment:
Ah, sorry my bad - didn't see that logic in the collapsed view 🙏 

Cohorts is upping its game :)